config/check_std_thread.cxx

```diff
@@ -0,0 +1,3 @@
+#include <thread>
+int main() {}
```

Comment:
Would it be possible to extend this test program to call `std::this_thread::yield();`?  Then, we could eventually re-use this for Linux (cf. issue #220).